package.json

```diff
@@ -20,6 +20,7 @@
     "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting"
   },
   "dependencies": {
+    "extend": "^3.0.0",
```

Comment:
You mean [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)? I don't think there is a native ES5/6 `Object.extend`, unless you're talking about Prorotype.js's [`Object.extend`](http://prototypejs.org/doc/latest/language/Object/extend/)

Regardless, no they don't, `Object.assign` is the closest native one, but it does not __deep-merge__ the objects, it's just shallow. I am looking for something like the [`jQuery.extend([deep, obj1, obj2, ... objN])`](https://api.jquery.com/jquery.extend/#jQuery-extend-deep-target-object1-objectN)